recirq/quantum_chess/quantum_board.py

```diff
@@ -343,6 +345,35 @@ def get_probability_distribution(self,
 
         return self.probabilities
 
+    def _generate_board_accumulations(self, repetitions: int = 1000) -> None:
+        """ Samples the state and generates the accumulated
```

Comment:
This language is bit confusing to me. Perhaps say "of each board in the state" instead of "the whole board"

Reply:
Thanks Chris. It's changed.